### ISSUE TITLE
I always load in as Franklin, and deleting the saved Franklin ped also deletes my saved ped

### ISSUE THREAD

**Issue by Hydraxonn** [Adopcalipt/RandomStart_Cplus#1]
Could be a conflict with another script I have, so I'll be testing some more soon. Whenever I load in, I always load in as Franklin. I have the Random Start option enabled, but then every random scenario disabled. This is because I have another mod that makes me load in at my last location. I want to load in as my saved ped in my last location every time I launch the game. But even after I save my custom ped, then relaunch the game, I still load in as Franklin and have another saved ped named Franklin. If I delete that saved Franklin ped, it also deletes my saved MP ped! I'd really appreciate it if this could be looked into, and if possible adding a "Load at Last Location" option would really help simplify things. Thanks for all your hard work on this awesome mod!

**Comment by Hydraxonn** [comment]
I should note that after I save my character, the INI for the ped DOES appear in /RandomStart/SavedPeds. an INI Does NOT appear for franklin, even though the menu option is there. If I relaunch my game without deleting either ped, I load in as Franklin, not my saved ped. I still have the Franklin AND my ped options in the menu.

**Comment by Adopcalipt** [comment]
which version are you running as this "Whenever I load in, I always load in as Franklin." should be fixed in latest version with the ped_lists that you can edit. also your main save character is made on load but not saved or part of the save ped_list however if you want them on your list you can save them in the ped edit menu, and they will add an name.ini to your saveped directory this is in the latest version previous version everything int the ped_list got saved causing blank ini's.

**Comment by Hydraxonn** [comment]
It's definitely creating its own ini file for the new saved ped, but Franklin still always appears in my saved peds list. Is this intentional? Just like the other issue I'd he happy to help continue testing

**Comment by Adopcalipt** [comment]
yes franklin, michael or trevor will be first in list except its not in the save list. this is so you can revert back to the main character if required it is stored separately and is not selected when random saved peds are loaded if you save any them there will be 2 references in the list on reload. also if possible can you add - this ; VER_1_0_2944_0, before VER_1_0_2944_0_STEAM, to the main.h to see if this is the fault with not recognizing your game version;


	VER_1_0_2944_0,

	VER_1_0_2944_0_STEAM,
	VER_1_0_2944_0_NOSTEAM,

**Comment by Hydraxonn** [comment]
I understand. That makes sense for why Franklin is always in my saved peds list. Maybe that could be better labelled, so that players understand why Michael Franklin or Trevor are always there at the top. The real bug, i guess, is that I can delete Franklin and it deletes my saved ped. That, and because Franklin is always at the top of my peds list I always load in as Franklin and not as my saved ped, even when the Saved Ped option is enabled.

As far as the version check is concerned, I'll add that line to main.h now and see if it fixes the game version issue. Thanks!

**Comment by Hydraxonn** [comment]
Yup, adding that line fixes the crash. Glad it's so simple! I can create a PR that adds this now.
